Content.Server/Revolutionary/Components/CommandStaffComponent.cs

```diff
@@ -1,11 +1,9 @@
-using Content.Server.GameTicking.Rules;
-
 namespace Content.Server.Revolutionary.Components;
```

Comment:
With these changes, perhaps the namespace should be changed.
It's kinda weird to be importing `Content.Server.Revolutionary...` stuff for non-Rev features.

Reply:
Content.Server.Roles.Jobs maybe?